models/vrp.rb

```diff
@@ -147,7 +147,7 @@ def self.check_consistency(hash)
         raise OptimizerWrapper::UnsupportedProblemError.new(
           'Services can appear in at most one shipment relation. '\
           'Following services appear in multiple shipment relations',
-          shipment_relations.detect{ |id| shipment_relations.count(id) > 1 }
+          [[shipment_relations.detect{ |id| shipment_relations.count(id) > 1 }]]
```

Comment:
I didn't verify this by the way, does it work as expected?

Reply:
It does